### PR TITLE
Clarified recommendation on 2nd submission round + some fine tuning

### PR DESCRIPTION
**Pull request by TeemuSailynoja**
most important finetune: added link to the Aalto Zoom quick start page.

### CODE REVIEW

File: assignments.Rmd [hunk 1]
```diff
@@ -34,9 +34,9 @@ individual assignments are not guaranteed to be up to date until Monday
 the deadlines below, e.g. the first assignment is not guaranteed to be up to
 date until 7.9 at 8am).
 
-Report all results in a single, anonymous *.pdf -file and submit it in
-[peergrade](https://www.peergrade.io). Submitting empty pdf or almost
-empty pdf is not allowed. Include also any source code to the report
+Report all results in a single, **anonymous** *.pdf -file and submit it in
+[peergrade](https://www.peergrade.io). Submitting empty or almost
+empty pdf files is not allowed. Include also any source code to the report
```
Comment (avehtari) [inline assignments.Rmd:39]: "file is" or "files are"

**Review by avehtari — CHANGES_REQUESTED** [review]
one typo to fix before merging

File: assignments.Rmd (hunk 1 above)
Comment (topipa) [reply assignments.Rmd:39]: I think the current form is correct. Submitting ... is not allowed, so it does not matter whether ... is plural or not

File: assignments.Rmd [hunk 2]
```diff
@@ -46,25 +46,26 @@ The assignments are mostly solved using computer (R or Python). Related
 demos for each assignment are available in the course web pages (links
 in Materials section). See [TA sessions](#TA_sessions) for getting help.
 
-For those who are self studying or who miss several assignment hand-ins
-for a good reason, there is possibility to hand-in all assignments once
-in January.
+For those who are self studying or who miss several assignment hand-ins, there
+is a possibility to hand-in all assignments once in January. We still strongly
```
Comment (topipa) [inline assignments.Rmd:50]: better could be "possibility to hand in"

File: assignments.Rmd [hunk 3]
```diff
@@ -125,31 +126,30 @@ The following definitely are reasons to react positively to feedback:
...
+Chat activity and the questionnaire are not included in the maximum assignment
+score and thus are not required to receive a full 100\% score.
 
 Other students' Chat activity will not affect one's resulting
-grade, i.e. there is no need to try to perform in the Chat or to take
-any part in it at all.
+grade, i.e. there is no need to try to perform in the Chat or to take any part in it at all.
 
-After each assignment, a voluntary anonymous questionnaire is opened on [the course page in mycourses](https://mycourses.aalto.fi/course/view.php?id=28239) for the peergrading feedback period - Monday to Wednesday 23:59.
+After each assignment, a voluntary anonymous questionnaire is opened on [the course page in mycourses](https://mycourses.aalto.fi/course/view.php?id=28239) for the peergrading feedback period (Monday to Wednesday 23:59).
```
Comment (topipa) [inline assignments.Rmd:136]: we could clarify "(Monday 8:00 to Wednesday 23:59)" to not be misunderstood as "(Monday 23:59 to Wednesday 23:59)"

**Review by topipa — COMMENTED** [review]
I added a few possible clarifications, but other than that everything looks good

File: assignments.Rmd (hunk 1 above)
Comment (avehtari) [reply assignments.Rmd:39]: ah, I was reading too quickly just the changed line and the whole sentence

**Pull request merged**